raiden-ts/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [Unreleased]
```

Comment:
In the bottom of the changelog files there's a line with the link to the github compare between versions, you need to bump them as well

Reply:
Ah, I see.